x-pack/plugins/siem/public/components/charts/common.tsx

```diff
@@ -63,6 +64,7 @@ export enum SeriesType {
   LINE = 'line',
 }
 
+// Customize colors: https://ela.st/custom-colors
```

Comment:
Adding a reference here to show an example from elastic-charts.
https://ela.st/custom-colors